airflow/bin/cli.py

```diff
@@ -1418,6 +1418,46 @@ def connections(args):
 
         return
 
+    if args.filepath:
+        with open(args.filepath, 'r') as stream:
+            connections_config = yaml.load(stream)
+        connections = {
+            connection['conn_id']: connection['conn_uri']
+            for connection in connections_config['connections']
+        }
+        # check conn_id and conn_uri were passed from yaml file.
+        missing_args = list()
+        for conn_id, conn_uri in connections.items():
```

Comment:
how about changing in host and port? https://airflow.lyft.net/connection/list/

Reply:
Here:https://github.com/lyft/incubator-airflow/blob/airflowinfra/airflow/models/connection.py#L119. Connection itself will parse the uri we provide and set host and port. So if other parts changed, we still can update the uri and let Connection to parse the update host and port.

Reply:
I am ok with it. But remember to provide an example in connections.yaml and explain that host, port, password, etc needs to be set in url.

Reply:
will do.